### DISCUSSION TITLE
[@types/google-publisher-tag] Missing setRequestNonPersonalizedAds

### DISCUSSION THREAD

**Discussion by sangxxh** [DefinitelyTyped/DefinitelyTyped#62895] [Issues with a @types package]
Please add `setRequestNonPersonalizedAds` to `PubAdsService` interface.

I have tried calling it in the example at https://developers.google.com/publisher-tag/guides/use-typescript and it does exist:

```
googletag.pubads().setRequestNonPersonalizedAds(1)
```

[a comment by an automated account is not shown]

**Comment by jimper** [comment]
(replying from the correct account this time)

That method is deprecated, you should use [PrivacySettingsConfig](https://developers.google.com/publisher-tag/reference#googletag.privacysettingsconfig) instead. See prior discussion @ https://github.com/googleads/google-publisher-tag-types/issues/2.
